### PR TITLE
Airbnb HW-Habtamu Abere

### PR DESCRIPTION
**Pull request by HabRonan**
**Deliverable Submission**

Please describe your comfort and completeness levels before submitting.

Comfort Level (1-5):  4.6

Completeness Level (1-5): 4.9

What did you think of this deliverable?: great practice, certainly need more practice

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,3 +1,183 @@
-h1 {
-  color: salmon;
+body{
+  font-family: "LL brown", Arial, sans-serif ;
```
Comment (amaseda) [inline css/style.css:2]: Did not know this was a font you could use w/o a CDN!

File: css/style.css [hunk 2]
```diff
@@ -1,3 +1,183 @@
...
+  display: inline-block;
+  text-decoration: none;
+  color:#000;
+  font-size: 20px;
+  padding-right: 20px;
+}
+
+/*extra link-Guidebooks and weekend*/
+
+div.Guidebooks>div{
```
Comment (amaseda) [inline css/style.css:67]: Common practice is not to capitalize CSS class names.

File: css/style.css [hunk 3]
```diff
@@ -1,3 +1,183 @@
...
+
+/*clear style*/
+.clearfix:after {
+        visibility: hidden;
+        display: block;
+        content: " ";
+        clear: both;
+        height: 0;
+        font-size: 0;
+      }
```
Comment (amaseda) [inline css/style.css:143]: Ideally, if you're using flex, you shouldn't need clearfix.

**Comment by amaseda** [comment]
:+1:

File: css/style.css (hunk 1 above)
Comment (HabRonan) [reply css/style.css:2]: yep I forgot to add the CDN, which is why the fonts are not that close to the homework.

File: css/style.css (hunk 2 above)
Comment (HabRonan) [reply css/style.css:67]: Yes, thanks. I usually focus first on the functionality and forget a lot of formatting and common practice. I guess I have to do it parallel.

File: css/style.css (hunk 3 above)
Comment (HabRonan) [reply css/style.css:143]: Oh, didn't knew that. the div after the one that is being flexed was following the top div that is getting flexed so I had to clear it.

**Pull request closed without merging**